public/locales/en/translation.json

```diff
@@ -159,7 +159,8 @@
   "pages": {
     "about": "About",
     "ethicalPrinciples": "Ethical principles",
-    "howDoesItWork": "How does it work?"
+    "howDoesItWork": "How does it work?",
+    "termsOfService": "Terms of Service"
```

Comment:
I added this to the `pages` key because all the other Contentful pages are listed here. Seemed the logical place.

I'm noticing now that there is a `tos` key already with that info in it. What are your thoughts on this?